cli/azd/internal/auth_type.go

```diff
@@ -6,9 +6,23 @@ type AuthType string
 const (
 	AuthTypeUnspecified AuthType = "UNSPECIFIED"
 	// Username and password, or key based authentication
-	AuthtypePassword AuthType = "PASSWORD"
+	AuthTypePassword AuthType = "PASSWORD"
 	// Connection string authentication
-	AuthtypeConnectionString AuthType = "CONNECTION_STRING"
+	AuthTypeConnectionString AuthType = "CONNECTION_STRING"
 	// Microsoft EntraID token credential
-	AuthtypeManagedIdentity AuthType = "MANAGED_IDENTITY"
+	AuthTypeManagedIdentity AuthType = "MANAGED_IDENTITY"
 )
+
+func GetAuthTypeDescription(authType AuthType) string {
+	switch authType {
+	case AuthTypeUnspecified:
+		return "Unspecified"
+	case AuthTypePassword:
+		return "Username and password"
+	case AuthTypeConnectionString:
+		return "Connection string"
+	case AuthTypeManagedIdentity:
```

Comment:
If we use umi here, maybe a const for umi, a const for smi?

Reply:
1. Rename to umi.
2. Not ass smi in current PR, we can add it when smi is used.